commit dd2b31dec0c0e792bec0486d72c8a049eea4bee1
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Sun Sep 3 02:45:06 2017 +0200

    hurd: Fix p{read,write}{,v64}v2.c build
    
            * sysdeps/posix/preadv2.c: Use off_t instead of OFF_T.
            * sysdeps/posix/preadv64v2.c: Use off_t instead of OFF_T.
            * sysdeps/posix/pwritev2.c: Use off_t instead of OFF_T.
            * sysdeps/posix/pwritev64v2.c: Use off_t instead of OFF_T.

diff --git a/sysdeps/posix/preadv2.c b/sysdeps/posix/preadv2.c
index d27f7028ed..b79b7b1bc3 100644
--- a/sysdeps/posix/preadv2.c
+++ b/sysdeps/posix/preadv2.c
@@ -23,7 +23,7 @@
 
 /* Since we define no flags for preadv2 just route to preadv.  */
 ssize_t
-preadv2 (int fd, const struct iovec *vector, int count, OFF_T offset,
+preadv2 (int fd, const struct iovec *vector, int count, off_t offset,
 	 int flags)
 {
   if (flags != 0)
diff --git a/sysdeps/posix/preadv64v2.c b/sysdeps/posix/preadv64v2.c
index ce7cb40bf2..ab71f027d4 100644
--- a/sysdeps/posix/preadv64v2.c
+++ b/sysdeps/posix/preadv64v2.c
@@ -20,7 +20,7 @@
 #include <sys/uio.h>
 
 ssize_t
-preadv64v2 (int fd, const struct iovec *vector, int count, OFF_T offset,
+preadv64v2 (int fd, const struct iovec *vector, int count, off_t offset,
 	    int flags)
 {
   if (flags != 0)
diff --git a/sysdeps/posix/pwritev2.c b/sysdeps/posix/pwritev2.c
index 7ec8cbc407..d746059d42 100644
--- a/sysdeps/posix/pwritev2.c
+++ b/sysdeps/posix/pwritev2.c
@@ -23,7 +23,7 @@
 
 /* Since we define no flags for pwritev2 just route to pwritev.  */
 ssize_t
-pwritev2 (int fd, const struct iovec *vector, int count, OFF_T offset,
+pwritev2 (int fd, const struct iovec *vector, int count, off_t offset,
 	  int flags)
 {
   if (flags != 0)
diff --git a/sysdeps/posix/pwritev64v2.c b/sysdeps/posix/pwritev64v2.c
index be98aeed9d..bf62d74190 100644
--- a/sysdeps/posix/pwritev64v2.c
+++ b/sysdeps/posix/pwritev64v2.c
@@ -21,7 +21,7 @@
 
 /* Since we define no flags for pwritev2 just route to pwritev.  */
 ssize_t
-pwritev64v2 (int fd, const struct iovec *vector, int count, OFF_T offset,
+pwritev64v2 (int fd, const struct iovec *vector, int count, off_t offset,
 	     int flags)
 {
   if (flags != 0)
